gpii/node_modules/transformer/test/js/TransformerTests.js

```diff
@@ -549,4 +549,239 @@ var fluid = fluid || require("infusion"),
+                        "to": "19:45"
+                    }
+                }
+            },
+            normalOrderMinutes: {
+                "foo": {
+                    "transform": {
+                        "type": "gpii.transformer.timeInRange",
+                        "inputPath": "bar",
+                        "from": "10:30",
+                        "to": "10:59"
+                    }
+                }
+            },
+            wrappedOrder: {
+                "foo": {
+                    "transform": {
+                        "type": "gpii.transformer.timeInRange",
+                        "inputPath": "bar",
+                        "from": "22:15",
+                        "to": "00:18"
+                    }
+                }
+            }
+        },
+        expects: {
+            "Basic hours - matches": {
+                rule: "normalOrder",
+                input: {
+                    bar: "11:00"
+                },
+                expected: {
+                    foo: true
+                }
+            },
+            "Basic hours - above threshold": {
+                rule: "normalOrder",
+                input: {
+                    bar: "19:47"
+                },
+                expected: {
+                    foo: false
+                }
+            },
+            "Basic hours - below threshold": {
+                rule: "normalOrder",
+                input: {
+                    bar: "10:28"
+                },
+                expected: {
+                    foo: false
+                }
+            },
+            "Minutes windows - matches": {
+                rule: "normalOrderMinutes",
+                input: {
+                    bar: "10:37"
+                },
+                expected: {
+                    foo: true
+                }
+            },
+            "Wrapped order - matches morning": {
+                rule: "wrappedOrder",
+                input: {
+                    bar: "00:12"
+                },
+                expected: {
+                    foo: true
+                }
+            },
+            "Wrapped order - matches evening": {
+                rule: "wrappedOrder",
+                input: {
+                    bar: "23:00"
+                },
+                expected: {
+                    foo: true
+                }
+            },
+            "Wrapped hours - above threshold": {
+                rule: "wrappedOrder",
+                input: {
+                    bar: "10:28"
+                },
+                expected: {
+                    foo: false
+                }
+            }
+        }
+    };
+
+    jqUnit.test("gpii.transformer.timeInRange tests", function () {
+        fluid.each(timeInRangeTests.expects, function (test, tname) {
+            // offset for timezone of tester to ensure that testfixture times are still valid
+            test.input.bar = test.input.bar + (new Date()).getTimezoneOffset() * 60000;
```

Comment:
thanks for catching this... It's a leftover from when the input was a timestamp.. I guess the reason the tests were still passing is due to this https://github.com/kaspermarkus/universal/blob/GPII-795/gpii/node_modules/transformer/src/js/Transformer.js#L168 apparently just parsing "47-6000000" to be the integer 47.. odd..